src/main/java/edu/miu/cs545/group1/MiniOnlineMarket/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,52 @@
+package edu.miu.cs545.group1.MiniOnlineMarket.service;
+
+import edu.miu.cs545.group1.MiniOnlineMarket.domain.Product;
+import edu.miu.cs545.group1.MiniOnlineMarket.domain.ProductReview;
+import edu.miu.cs545.group1.MiniOnlineMarket.repository.ProductRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+
+    @Autowired
+    ProductRepo productRepo;
+
+    @Override
+    public List<Product> findAll() {
+        return productRepo.findAll();
+    }
+
+    @Override
+    public Product findById(Long id) {
```

Comment:
@CitizenOneEt could we handle the scenario where no item is returned i.e. null. It might throw an exception

We could do this so it return null when no product is found

```
   public Product findById(Long id) {
        return productRepo.findById(id).orElse(null);
    }
```